src/main/resources/script_templates/Deconvolution/ExtractPSF.py

```diff
@@ -0,0 +1,62 @@
+# @OpService ops
+# @ImgPlus beads
+# @OUTPUT ImgPlus thresholded
+# @OUTPUT ImgPlus points
+# @OUTPUT ImgPlus psf
```

Comment:
With the new language-agnostic parameter syntax, you can write:
```
#@ OUTPUT ImgPlus psf
```
(i.e. _without_ space between `#@`, and with any number of spaces before the type declarations, and anywhere in the script, not restricted to the header), the advantage being that it's the same across all script languages.